start_bundle_coupled.R

```diff
@@ -5,8 +5,30 @@
 # |  AGPL-3.0, you are granted additional permissions described in the
 # |  REMIND License Exception, version 1.0 (see LICENSE file).
 # |  Contact: remind@pik-potsdam.de
-library(stringr)
 require(lucode2)
+require(magclass)
+require(gms)
+require(remind2)
+require(gtools) # required for mixedsort()
+require(dplyr) # for filter, secelt, %>%
+require(stringr) # for str_sub
+
+helpText <- "
+#' Usage:
+#' Before running this script, edit this file and provide user settings
```

Comment:
What is "this file" here?

Reply:
well, `start_bundle_coupled.R` itself

Reply:
I agree `--test` is not entirely appropriate, but: We had a `test` mode in `start_bundle_coupled.R` since Lavinia [moved the code to github](https://github.com/remindmodel/remind/blob/6dc75ecdfc61da2e4c439fc2f3e8ce0c6d3d81f4/start_bundle_coupled.R#L63), and it simply [did not submit](https://github.com/remindmodel/remind/blob/6dc75ecdfc61da2e4c439fc2f3e8ce0c6d3d81f4/start_bundle_coupled.R#L321-L322) the runs to the cluster.

I liked that and [added such a mode to start.R in March](https://github.com/remindmodel/remind/pull/701). So I guess this naming is already well established so I would rather not dare to change it.

Reply:
Alright, then --test is the better choice, I agree. I was thinking if another script for some reason runs `source("start_bundle_coupled.R")` it might be confusing to have "this file". You could consider just writing out "start_bundle_coupled.R", but I approve either way :slightly_smiling_face:

Reply:
I fixed the `--help` text. For `output.R`, one may think in the future to pass the `--test` option to the individual output scripts (in `direct` mode) and implement it such that they don't write stuff, but rather test whether everything is fine. But this would be quite some work, I guess.